install/kubernetes/helm/istio/charts/tracing/templates/service.yaml

```diff
@@ -38,4 +42,5 @@ items:
         targetPort: {{ .Values.service.uiPort }}
     selector:
       app: jaeger
+    type: {{ .Values.service.uiType }}
```

Comment:
We do not want to expose service type here, and intend to use Gateway to expose those services. 

Suggest your PR focus on adding `annotations` but do not touch service type in this PR

Reply:
Just removed any changes that are unrelated to the annotations.